app/views/documents/show/_historical_notice.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= render "govuk_publishing_components/components/notice", {
+  title: I18n.t("documents.show.historical.title",
+                document_type: @edition.document_type.label.downcase),
+  description_text: I18n.t("documents.show.historical.description",
```

Comment:
This alignment seems a bit off to me - I suppose it's to keep the line length below 80 - would it be neater if everything were in its own JSON-style block?

```ruby
{
  document_type: @edition.document_type.label.downcase),
  description_text: I18n.t("documents.show.historical.description",
  government_name: @edition.government.name),
}
```

Reply:
Will adjust, thanks :) and thanks for thinking it was something to do with linting, it's actually an oversight as the linter doesn't check these files. Will go with your interpretation though, makes me look better.